app/assets/javascripts/rad_common/dynamic_updater.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready( function() {
```

Comment:
This is a minimal approach to the pattern we have used in the past to update ajax fields. We can expand on it later to make it work in more situations, but main idea here is not to have to add much extra html and/or controller code for this to work out of the box